components/discord_webhook/actions/send-message-advanced/send-message-advanced.mjs

```diff
@@ -46,13 +46,21 @@ export default {
       content += `\n\n${this.getSentViaPipedreamText()}`;
     }
 
-    // No interesting data is returned from Discord
-    await this.discordWebhook.sendMessage({
+    const params = {
       avatarURL,
       embeds,
       content,
       threadID,
       username,
-    });
+    };
+
+    try {
+      // No interesting data is returned from Discord
+      await this.discordWebhook.sendMessage(params);
+      $.export("$summary", "Message sent successfully");
+    } catch (err) {
```

Comment:
Re-throw the caught `err` (or a [formatted error](https://github.com/PipedreamHQ/pipedream/blob/55236b3aa993cbcb545e245803d8654c6358b0a2/components/airtable/airtable.app.js#L71)) so the workflow job fails and the error gets sent to system notifications or the global error workflow?

Reply:
Addressed